src/main/webapp/search-script.js

```diff
@@ -1,8 +1,8 @@
 function collectUnitsFromSearch() {
```

Comment:
ok maybe this is an issue that will go away when I push? Maybe this is something prettier is doing